spring-grpc-core/src/main/java/org/springframework/grpc/client/DefaultGrpcChannelFactory.java

```diff
@@ -46,16 +48,16 @@ public class DefaultGrpcChannelFactory implements GrpcChannelFactory, Disposable
 
 	private final List<GrpcChannelConfigurer> configurers = new ArrayList<>();
 
+	private final ClientInterceptorsConfigurer interceptorsConfigurer;
+
 	private ChannelCredentialsProvider credentials = ChannelCredentialsProvider.INSECURE;
 
 	private VirtualTargets targets = VirtualTargets.DEFAULT;
 
-	public DefaultGrpcChannelFactory() {
-		this(List.of());
-	}
-
-	public DefaultGrpcChannelFactory(List<GrpcChannelConfigurer> configurers) {
+	public DefaultGrpcChannelFactory(List<GrpcChannelConfigurer> configurers,
+			ClientInterceptorsConfigurer interceptorsConfigurer) {
```

Comment:
I added to `<init>` as it is required if user wants any kind of interceptors. I see that you did `targets` and `credentials` w/ setters. I am guessing to avoid constructor bloat and possible to let users override if need be? Should we do the same for this CIC?